packages/hyperion-flowlet/src/global.d.ts

```diff
@@ -6,4 +6,5 @@
 
 interface GlobalFlags {
   preciseTriggerFlowlet?: boolean;
-}
\ No newline at end of file
+  useNewIgnoreInteractableImpl?: boolean;
```

Comment:
Don't add this flag in this package. Add it in the `hyperion-autologging` package. The mechanism support augmenting the type globally. 
You should always assume each package being independent and separate from others.